### ISSUE TITLE
FTBased EventBuilderFilter

### ISSUE THREAD

**Issue by dglazier** [JeffersonLab/iguana#202]
Need an option, or a separate filter to work on FTBased PID from RECFT::Particle::pid
If no FT particle exists, should default to REC::Particle for that event.

**Comment by c-dilks** [comment]
FTEnergyCorrection and its validator will also need this sort of feature

**Comment by dglazier** [comment]
Not sure about that. FTEnergyCorrection should apply to any charged particle in the FT. The FTBased pid does not effect that.

**Comment by dglazier** [comment]
Yes it should change both. Although if someone is using an electron in the FT and not using FT based, it is likely they are doing something wrong.
